src/Models/BaseElement.php

```diff
@@ -579,7 +579,7 @@ public function getPage()
             return $this->cacheData['page'];
         }
 
-
+        // Despite the name of the method, method can return any type of DataObject
```

Comment:
Which method are you talking about here? If you mean `getPage`, use the suggestion below - otherwise use the name of whatever method you're talking about.

Also, move the comment into the PHPDocs for the method instead, so developers can take advantage of their IDE displaying it to them. You should also update the typing mentioned in the PHPDoc since at the moment it explicitly says `null|SiteTree`.
```suggestion
        // Despite the name of the method, getPage can return any type of DataObject
```